instrumentation/httpclient-jdk11/src/main/java/com.nr.agent.instrumentation.httpclient/Util.java

```diff
@@ -0,0 +1,99 @@
+package com.nr.agent.instrumentation.httpclient;
+
+import com.newrelic.agent.bridge.AgentBridge;
+import com.newrelic.api.agent.*;
+import com.newrelic.api.agent.weaver.Weaver;
+
+import java.net.ConnectException;
+import java.net.URI;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.concurrent.CompletionException;
+import java.util.function.BiConsumer;
+import java.util.logging.Level;
+
+public class Util {
```

Comment:
Might not be a big deal, but we might want to qualify our "Util" classes with a semi-unique prefix so when searching in IntelliJ its easier to resolve.  Once again, maybe not a big deal for our work but in other projects I've been on it was.

Reply:
Sounds good. What do you suggest here?

Reply:
Once again, not suggesting you change now after the fact, but if you think its helpful in the future `SomethingSpecificUtil` might make it easier to find and distinguish.